pkg/dump/dump.go

```diff
@@ -28,6 +28,10 @@ type Config struct {
 	// tags.
 	SelectorTags []string
 
+	// LookUpSelectorTagsConsumers can be used to lookup entities using other
```

Comment:
Can you please expand on this? As it is, it's not immediate how this can be used and when.

Reply:
@GGabriele for instance CMA is creating consumers using a pipeline 1 with tag "consumers_global". 
They have a pipeline 2 for services (tag: serviceName) that could for instance set up "rate-limting" plugins on different consumers, as these consumers are not part of the sync for this pipeline it will fail.

Using a LookUpSelectorTagsConsumers you can specify deck to check if the consumers exist on a different tag

Please let me know if you have other questions